app/assets/javascripts/backbone/views/breakeven_product_total_view.coffee

```diff
@@ -0,0 +1,104 @@
+class MS.Views.BreakevenProductTotalView extends Backbone.View
+
+  # The view is called from the editiable table view.
+
+  # events:
+  #   'click tr [data-action="delete"]': 'removeRow'
+  #   'click .actions [data-action="add"]': 'addRow'
+
+  initialize: (options) ->
+    @products = options.products
+
+    @totalFixedCosts = parseFloat(@$('.total_fixed_costs').val())
+
+    _.each @products, (product) =>
+      product.on "product:changed", () =>
```

Comment:
Thanks for reminding me. As for the even hash, I was unaware you can use it to attach events to non-dom objects. What is the syntax to do that? Google was unhelpful for me.

I changed it to: 
```coffeescript
@products.on 'product:changed', =>
  @calculateTotals
```